config/configmapprovider/provider.go

```diff
@@ -77,3 +79,23 @@ type ChangeEvent struct {
 	// Any non-nil error indicates that there was a problem with watching the config changes.
 	Error error
 }
+
+// Retrieved holds the result of a call to the Retrieve method of a Provider object.
+type Retrieved struct {
+	Cfg *config.Map
```

Comment:
```suggestion
	Map *config.Map
```

Not fond of the `Cfg` field name.

Reply:
Done.